src/components/Profile/ProfileComponent.jsx

```diff
@@ -107,6 +108,11 @@ const ProfileComponent = ({ contactProfile, webId }) => {
             gap: '10px'
           }}
         >
+          <ProfileEditButtonGroup
+            edit={edit}
+            handleCancelEdit={handleCancelEdit}
+            handleEditInput={handleEditInput}
+          />
```

Comment:
The component that shows the buttons for updating profile information should only be available to the user. They should not have access to this button group in other people's profiles. That was the reason behind the component being inside the condition `!contactProfile`.

If we want to move this out of the box below, make it render with the same condition similar to the invite link.